data/.directory

```diff
@@ -0,0 +1,4 @@
+[Dolphin]
```

Comment:
What is this file for? I don't think `bambam` uses it? In that case please add it to `.gitignore` and remove from PR.